react/src/views/Transfers/CreateTransferShipment/CreateTransferShipmentView.test.tsx

```diff
@@ -0,0 +1,235 @@
+import "@testing-library/jest-dom";
+import { screen, render, cleanup } from "tests/test-utils";
+import { organisation1 } from "mocks/organisations";
+import { acceptedTransferAgreement } from "mocks/transferAgreements";
+import userEvent from "@testing-library/user-event";
+import { assertOptionsInSelectField, selectOptionInSelectField } from "tests/helpers";
+import { GraphQLError } from "graphql";
+import CreateTransferShipmentView, {
+  ALL_ORGS_AND_BASES_WITH_TRANSFER_AGREEMENTS_QUERY,
+  CREATE_SHIPMENT_MUTATION,
+} from "./CreateTransferShipmentView";
+
+jest.setTimeout(30000);
```

Comment:
This is scary. Why is this needed?

Reply:
@HaGuesto It resolves the random timeout issue when CircleCI runs the tests, but we can remove it locally. If you have any other ideas, please let me know.